packages/@aws-cdk/cloud-assembly-schema/lib/assembly-manifest.ts

```diff
@@ -0,0 +1,340 @@
+interface BaseAssetMetadataEntry {
+    /**
+     * Requested packaging style
+     */
+    readonly packaging: string;
+
+    /**
+     * Logical identifier for the asset
+     */
+    readonly id: string;
+
+    /**
+     * The hash of the source directory used to build the asset.
+     */
+    readonly sourceHash: string;
+
+    /**
+     * Path on disk to the asset
+     */
+    readonly path: string;
```

Comment:
Wonder if this should be renamed to `uri` since path is very filesystem specific (tsdoc also says 'on disk') while there is a separate `FileAssetMetadataEntry` that represents a file based entry. 
OTOH, we move this into `FileAssetMetadataEntry`? 

I am aware that this is probably mostly cut-paste from existing cx-api, but maybe it's an excellent opportunity to clean this a little?

Reply:
I hear you, but we can't do that. This maps to the `manifest.json` files, so the property names must be preserved to support reading old manifest files. Unless we implement a customer json parser, but, yeah...

Reply:
Ok, let's leave it as is.